web/pages/base_page.rb

```diff
@@ -4,6 +4,35 @@ module Web
   module Pages
     class BasePage <SitePrism::Page
       element :logo, 'a.header__logo'
+      element :search_field, 'input.search-form__input'
+      element :search_button, 'button.search-form__submit'
+      element :clear_button, 'button.search-form__clear'
+      elements :suggestion_items, 'ul.suggest-list li.search-suggest__item'
+      element :empty_results, "div.search-nothing div h1"
+      element :search_suggest_no_results_msg, "p.search-suggest__item-content_type_no-results"
+      element :search_suggest, 'div.search-suggest p.search-suggest__item-content_type_no-results'
```

Comment:
the same as above and also, looks like we are accesing the same element locator here:
`p.search-suggest__item-content_type_no-results`